core/config/constants.js

```diff
@@ -52,14 +52,13 @@ const throttling = {
 /**
  * @type {Required<LH.SharedFlagsSettings['screenEmulation']>}
  */
-const MOTOG4_EMULATION_METRICS = {
+const MOTOGPOWER_EMULATION_METRICS = {
   mobile: true,
-  width: 360,
-  height: 640,
-  // Moto G4 is really 3, but a higher value here works against
-  // our perf recommendations.
+  width: 412,
+  height: 767,
```

Comment:
did this include when you scrolled and the omnibox was offscreen? (i'm guessing so cuz this is actually a comparatively skinny aspect ratio)

767 feels kinda awk to me.  but i looked at the phones in CDT's emulated devices.. their aspect ratios... heights.. And tbh i dont think i can make a strong argument to change the number.  So yah. sg.

![image](https://user-images.githubusercontent.com/39191/212240418-4f9614f3-1bd5-4410-8e49-1a04b18e7e19.png)

Reply:
> did this include when you scrolled and the omnibox was offscreen? (i'm guessing so cuz this is actually a comparatively skinny aspect ratio)

No, the omnibox is on the screen.

<img width="1380" alt="image" src="https://user-images.githubusercontent.com/4071474/216128209-ffc7a834-e5a9-4d93-9d87-5d68cb2a7bd6.png">

Reply:
You should test on a page with a scrollable content and scroll down a bit to get the omnibox offscreen. Im pretty sure the other device preset heights are consistent with this.

Reply:
ok, that gets me 412x823